README.md

```diff
@@ -220,22 +222,46 @@ If `<condition>` or one of the `<expression>`s do not fit onto one line:
  else <expression>)
 ```
 
+### Case-statements
+
+```Ada
+case <selector>
+   when <first_alternative>  => <statement>;
+   when <second_alternative> => <statement>;
+   when others               => <statement>;
+end case;
+```
+
+If at least one `<statement>` does not fit onto the line or at least one alternative has multiple statements, *all* lines are wrapped and arrows are *not* aligned:
+
+```Ada
+case <selector>
+   when <first_alternative> =>
+      <statement>;
+      <statement>;
+   when <second_alternative> =>
+      <statement>;
+   when others =>
+      <statement>;
+      <statement>;
+end case;
+```
 ### Case-expressions
 
 ```Ada
 (case <selector>
- when <alternative> => <expression>,
- when <alternative> => <expression>,
- when others        => <expression>)
+ when <first_alternative>  => <expression>,
+ when <second_alternative> => <expression>,
+ when others               => <expression>)
```

Comment:
Why do we use another indentation as for case statements here? The current style for case expressions is very badly readable in my opinion.

Reply:
I agree that we should not have two variants and that the readability of case expressions is not great. We could unify case expressions as follows:

```Ada
(case <selector>
    when <first_alternative>  => <expression>,
    when <second_alternative> => <expression>,
    when others               => <expression>)

(case <selector>
    when <first_alternative> =>
       <expression>,
    when <second_alternative> =>
       <expression>,
    when others =>
       <expression>)
```

@jklmnn @treiher What do you think?

Reply:
I'm all for it.